### PR TITLE
Null field check to eliminate false positives

### PR DESCRIPTION
**Pull request by tatsu-i**
It was corrected with reference to this issue
https://github.com/Neo23x0/sigma/issues/777

### CODE REVIEW

File: rules/windows/builtin/win_alert_ad_user_backdoors.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ logsource:
 detection:
     selection1:
         EventID: 4738
+    filter_null:
+        AllowedToDelegateTo: null
     filter1:
         AllowedToDelegateTo: 
             - null
```
Comment (Neo23x0) [inline rules/windows/builtin/win_alert_ad_user_backdoors.yml:26]: The filter that you try to add should already be active with the expression in line 26

Comment (Karneades) [reply rules/windows/builtin/win_alert_ad_user_backdoors.yml:26]: This is what I meant in #777. I think the ES backend has troubles converting the list entry into the correct not-null expression.

**Comment by Neo23x0** [comment]
Could you then remove the other expression? 
Screenshot 2020-05-19 at 14 48 43

**Comment by Neo23x0** [comment]
I removed the other expression in `filter1`, which is then redundant, isn't it?

**Pull request merged**